### ISSUE TITLE
Kevin - Thank you for humoring me - please close after reading

### ISSUE THREAD

**Issue by rbucker** [wycats/handlebars.js#486]
After reading a number of issues and posts there seemed to be a philosophy of keeping handlebars as simple as possible and yet functional. I recently posted a request for a #join or a #each with a separator because I was essentially refactoring my JavaScript container object from the backend worker to the frontend client... and I did not want to cut/paste between object schemas so that it would fit.

_I have a great working solution which you provided; it is the correct balance._

That said, I read [this article](http://goo.gl/44lhi). In item (4) the author is recommending that the rendering take place on the client-side. Which I think we all agree with. The challenge is, however, as I described it previously. What sort of tools do we have to refactor a JavaScript object; whether it's to redact certain data or simply pivot it so that the template tool can render it properly without add so much code or system resources that it would have been better to render it server-side.

In conclusion I'm not certain whether this is the right question, talking point, or forum... but since you guys are the template experts it seemed like a good place to start. 

(issues databases are not the place to have conversations; in the absence of your personal email address)

Thanks again.
